packages/build-info/src/runtime/erlang.ts

```diff
@@ -0,0 +1,7 @@
+import { LangRuntime } from './runtime.js'
+
+export class Erlang extends LangRuntime {
+  id = 'erlang'
+  name = 'Erlang'
+  configFiles = ['rebar.config', 'rebar.lock']
```

Comment:
My quick research says a nontrivial fraction of Erlang projects won't be using Rebar. As long as that's ok, this seems fine 🤷🏼‍♂️.

Reply:
Yes, I meant for this to serve as a starting point for us. I don't personally know of any other markers to check for, but if you happen to know any, let's add them!